internal/daemonset/daemonset.go

```diff
@@ -409,28 +411,29 @@ func MakeUnloadCommand(spec kmmv1beta1.ModprobeSpec, modName string) []string {
 		"-c",
 	}
 
-	unloadCommand := "modprobe"
+	var unloadCommand strings.Builder
+	unloadCommand.WriteString("modprobe")
 
 	if ra := spec.RawArgs; ra != nil && len(ra.Unload) > 0 {
-		unloadCommand = fmt.Sprintf("%s %s", unloadCommand, strings.Join(ra.Unload, " "))
-		return append(unloadCommandShell, unloadCommand)
+		fmt.Fprintf(&unloadCommand, " %s", strings.Join(ra.Unload, " "))
```

Comment:
```suggestion
	for _, a := range ra.Unload {
		unloadCommand.WriteRune(' ')
		unloadCommand.WriteString(a)
	}
```

Reply:
I think this also applies in other places.

Reply:
Indeed, it applies to other places. Amended commit to cover them.